### PR TITLE
[AL] Add matriz de mudança de coordenadas (aula 18)

### PR DESCRIPTION
**Pull request by itsVugz**

### CODE REVIEW

File: content/al/0018-matriz-mudanca-de-coordenadas.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+:::tip[Definição]
+
+Sejam $B_1=\{v_1,...,v_n\}$ e $B_2=\{w_1,...,w_n\}$, duas bases ordenadas de um espaço vetorial $V$,
+a matriz $S_{B_1 \to B_2} = (s_{ij})_{n\times n}$, cujas colunas são as coordenadas dos vetores de $B_1$ escritos na base $B_2$,
+chama-se matriz de mudança de coordenadas de $B_1$ para $B_2$.
+
+:::
+
+:::tip[Teoremas]
```
Comment (diogotcorreia) [inline content/al/0018-matriz-mudanca-de-coordenadas.md:25]: I don't think this should be a container at all, just regular text

File: content/al/0018-matriz-mudanca-de-coordenadas.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+$$
+
+Sendo aplicada a matriz de mudança de coordenadas $S_{B_2 \to B_3}$ após a matriz de mudança de coordenadas $S_{B_1 \to B_2}$,
+podemos obter a matriz de mudança de coordenadas $S_{B_1 \to B_3}$.
+
+:::
+
+## Construção da matriz de mudança de coordenadas
+
+:::tip[Resolução]
```
Comment (diogotcorreia) [inline content/al/0018-matriz-mudanca-de-coordenadas.md:60]: I don't think this should be a container at all, just regular text

**Review by diogotcorreia — APPROVED** [review]
I'm gonna approve because it's not very relevant, but I left two comments that may need to be discussed. Let me know if I should merge.

File: content/al/0018-matriz-mudanca-de-coordenadas.md (hunk 2 above)
Comment (itsVugz) [reply content/al/0018-matriz-mudanca-de-coordenadas.md:60]: Agreed

File: content/al/0018-matriz-mudanca-de-coordenadas.md (hunk 1 above)
Comment (itsVugz) [reply content/al/0018-matriz-mudanca-de-coordenadas.md:25]: Agree

**Pull request merged**